macros/edr/tests/test_utils/get_test_type.sql

```diff
@@ -0,0 +1,42 @@
+{% macro get_test_type(flattened_test) %}
+    {% if flattened_test.test_namespace == "elementary" %}
+        {%- set elementary_test_type = get_elementary_test_type(flattened_test) %}
+    {% endif %}
+
+    {%- if elementary_test_type and elementary_test_type != "python_test" %}
```

Comment:
I know python test is not dbt test, but we save this type to the test results. 
I wanted to maintain the current flow that saves these as dbt tests. 
WDYT?